extendr-api/src/scalar/rcplx.rs

```diff
@@ -0,0 +1,186 @@
+use crate::scalar::macros::*;
+use crate::scalar::Rfloat;
+use crate::*;
+use std::convert::TryFrom;
+use std::ops::{Add, Div, Mul, Neg, Sub};
+use std::ops::{AddAssign, DivAssign, MulAssign, SubAssign};
+
+// #[cfg(feature="num-complex")]
```

Comment:
Should this be uncommented?

Reply:
The next step is to make rcplx_full.rs and rcplx_default.rs

These will be two alternative implementations with varying levels of support.

Reply:
Note that num-complex also supports a wide range of operations on complex numbers such as logs,
trig, arg and magnitude.